main.tf

```diff
@@ -3,8 +3,8 @@ data "aws_vpc" "vpc" {
 }
```

Comment:
Please update readme and add tag to latest current version. Please issue new tag 3.0.0 after merging to master. New version with `v5.0` provider should be added as version requirement.